packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1412,6 +1412,7 @@ export class ContainerRuntime
 			eventName: "ContainerLoadStats",
 			...this.createContainerMetadata,
 			...this.dataStores.containerLoadStats,
+			dmInitialSeqNumber: this.deltaManager.initialSequenceNumber,
```

Comment:
Container:Load_end will have sequenceNumber property. That should be enough, no?

Reply:
Likely to contain what we need. @agarwal-navin @NicholasCouri @markfields fyi

Reply:
Yeah, looks like it. sequenceNumber kind of makes sense. It is the sequence number that has been processed when container load completes.

Reply:
Container:Load_end reports a bunch of stuff:
- versionId, though I do not remember if it's a handle or not
- sequenceNubmer of summary we loaded from
- current sequenceNumber (in case we processed ops on top of snapshot).

So we should have complete picture.

Reply:
Closing this PR